tests/model_inheritance/tests.py

```diff
@@ -191,6 +192,20 @@ def b():
                     sql = query["sql"]
                     self.assertIn("INSERT INTO", sql, sql)
 
+    def test_create_copy_with_inherited_m2m(self):
+        restaurant = Restaurant.objects.create()
+        supplier = CustomSupplier.objects.create(
+            name="Central market", address="944 W. Fullerton"
+        )
+        supplier.customers.set([restaurant])
+        old_customers = supplier.customers.all()
+        supplier.pk = None
+        supplier.id = None
```

Comment:
This assignment seems redundant with the one above?

Reply:
I wanted to use [the documented way](https://docs.djangoproject.com/en/4.1/topics/db/queries/#copying-model-instances) of copying instances.